docs/docs/index.md

```diff
@@ -134,6 +134,38 @@ with q.connection.pipeline() as pipe:
 
 `Queue.prepare_data` accepts all arguments that `Queue.parse_args` does.
 
+### Batching jobs
+_New in version 1.16.0._  
+Multiple jobs can be added to a Batch to allow them to be tracked by a single ID:
+
+```python
+from rq.batch import Batch
+
+batch = Batch.create(id="my_batch", connection=redis_conn)
+jobs = q.enqueue_many(
+  [
+    Queue.prepare_data(count_words_at_url, ('http://nvie.com',), job_id='my_job_id'),
+    Queue.prepare_data(count_words_at_url, ('http://nvie.com',), job_id='my_other_job_id'),
+  ],
+  batch=batch  # You can also pass the batch ID as a string, i.e., batch="my_batch"
```

Comment:
I think the interface we expose should be `batch.enqueue_many()`

Reply:
@selwin I've moved the enqueueing logic to the batch class to match that interface.